docs/detections/alerts-ui-manage.asciidoc

```diff
@@ -37,6 +37,26 @@ image::images/additional-filters.png[Alerts table with Additional filters menu h
 
 * View detection alerts generated by a specific rule. Go to *Manage* -> *Rules*, then select a rule name in the table. The rule details page displays a comprehensive view of the rule's settings, and the Alerts table under the Trend histogram displays the alerts associated with the rule, including alerts from any previous or deleted revision of that rule.
 
+[float]
+[[group-alerts]]
+=== Group alerts
+
+beta:[] You can group alerts by rule name, host name, user name, source IP address, or any other field. Select *Group alerts by*, then select an option or *Custom field* to specify a different field.
```

Comment:
Is this `beta:[]` tag correct? It looks weird rendered: 

<img width="640" alt="Screenshot 2023-03-28 at 8 07 31 AM" src="https://user-images.githubusercontent.com/6935300/228265181-4fdbc3d6-216c-47cc-af23-b365ce0da7cd.png">